meetings/agenda.md

```diff
@@ -12,3 +12,32 @@ Please file pull requests to add, or discuss items to add, to the agenda.
 [Maximilian Bösing](https://github.com/boesing) wants to drop the line length limit from the `laminas-codestandard` ruleset.
 
 The line length limit is counter-productive in combination with psalm and more precise argument-/return-type definitions.
+
+### Allow PHP-Assert along with Webmozart-Assert in Specific Cases
+
+[Maximilian Bösing](https://github.com/boesing) wants to talk about the usage of `assert` (PHP) and `Assert` (Webmozart) when providing types for psalm.
+In some cases, having runtime assertions may consume unnecessary opcodes just to provide proper psalm-types.
+
+Lets assume the following code:
+
+```php
+$foo = sprintf('foo bar baz %d', 1);
+assert($foo !== '');
+```
+
+Psalm is not able to verify that `$foo` is a non-empty-string, even tho, the first argument is a non-empty string and therefore, the return value must either be `false` (in case of an error) or `non-empty-string`.
```

Comment:
I think this is potentially a bug: if you do an `assert(! empty($foo))` then it works here

Reply:
It does with `assert($foo !== '');` as well. But without `assert` psalm thinks its just a string and thus, the option would be either to use `@var string-not-empty`, `Assert::stringNotEmpty` or `assert()`.

The main discussion would be if we want to allow `assert` in components or not. I had a PR where for the sake of consistency with other packages, `Assert` had to be used. But `Assert` has runtime impact on production systems and I actually would prefer not having to do runtime assertions just to provide psalm an information which it should be able to infer.